geomesa-tools/bin/common-functions.sh

```diff
@@ -344,6 +344,41 @@ function geomesaScalaConsole() {
   exec $scalaCMD ${OPTS} -classpath ${classpath} -i "${GEOMESA_CONF_DIR}/.scala_repl_init"
 }
 
+function isExportLeaflet() {
+  params=($@)
+  for i in ${params[@]}; do
+    if [[ "${i}" == "leaflet" || "${i}" == *html ]]; then
```

Comment:
could we make this check both the flag and the value, to be more robust? this syntax is totally wrong but like: `if [[ "($params[i]" == "leaflet" && ($params[i-1] == "-F" || $params[i-1] == "--format")) || /* check output file */]]`